src/deploy/campaign/environments.ts

```diff
@@ -113,8 +125,9 @@ export const getConfig = async ({
       defaultRoyaltyFraction: royaltyFraction,
     },
     rootPriceConfig: priceConfig,
+    zTokenConfig: zConfig,
     zeroVaultAddress: zeroVaultAddressConf as string,
-    mockMeowToken: process.env.MOCK_MEOW_TOKEN === "true",
+    mockZToken: !!process.env.MOCK_Z_TOKEN,
```

Comment:
this is not correct. this variable comes in as a string, NOT as a boolean. so adding "!!" here will make it `true` if it's any string other than empty string.
instead you should write it as `process.env.MOCK_Z_TOKEN === "true"`. here you are comparing the string you read from the env to the exact string you need.
remeber that string and boolean are very different types!